core/silkworm/consensus/base/engine.cpp

```diff
@@ -235,4 +230,26 @@ std::optional<intx::uint256> EngineBase::expected_base_fee_per_gas(const BlockHe
     }
 }
 
+evmc::bytes32 EngineBase::compute_transaction_root(const BlockBody& body) {
+    if (body.transactions.empty())
```

Comment:
Would be nice to keep curly braces for `if` blocks

Reply:
done